.github/ISSUE_TEMPLATE/5.bug_report.md

```diff
@@ -1,17 +1,17 @@
 ---
```

Comment:
should we just remove this in favor of the new report template?

Reply:
The new templates don't render when initiated from the GH mobile app yet. So that option will still be presented as a fallback.

I think keeping it and phasing out once the the forms feature has that support.